packages/core/src/payment/strategies/paypal-commerce/paypal-commerce-payment-strategy.ts

```diff
@@ -72,6 +73,8 @@ export default class PaypalCommercePaymentStrategy implements PaymentStrategy {
             initializationMethodId = methodId;
         } else if (this._isPaylater) {
             initializationMethodId = 'paylater';
+        } else if (isVenmo && isVenmoEnabled) {
```

Comment:
Is there a case where the data will be `isVenmoEnabled` is false and `isVenmo` is true, or vice versa?

Reply:
Yes, there is such a case when the Venmo experiment is disabled.